src/v3/src/components/DuoWindow/DuoWindow.tsx

```diff
@@ -43,10 +43,14 @@ const DuoWindow: UISchemaElementComponent<{
         sig_request: signedToken,
         iframe: 'duo_iframe',
 
-        // @ts-expect-error type mismatch on post_action
-        post_action: (signedData: string) => {
```

Comment:
FYI, G2 uses a customized version of the duo web sdk that takes this post_action callback and returns a different value than what comes OOB with this SDK. The submit_callback is what is available out of the box and only requires us to parse the form element in the callback to retrieve the signature data.